test/fixtures/discovery/sap_system/expected_published_sap_system_discovery_application.json

```diff
@@ -0,0 +1,245 @@
+{
+  "agent_id": "779cdd70-e9e2-58ca-b18a-bf3eb3f71244",
+  "discovery_type": "sap_system_discovery",
+  "payload": [
```

Comment:
This is intended to be a list of objects instead of a single object instance if I remember how the sap system discovery works. Is this the case?

Reply:
it is always a list of objects